deploy/cloud/variables.tf

```diff
@@ -0,0 +1,54 @@
+## Deployment configuration
+
+variable "ess_region" {
+  default     = "gcp-us-central1"
```

Comment:
Usually, `gcp-us-west2` and `gcp-europe-west1` is the regions where all of the BCs are being published, maybe we should use them instead.

Reply:
It depends on the "default" flow. 
If we want to `terraform apply` do we want to create a snapshot version to the latest ga?

Reply:
they also contain `SNAPSHOT`
<img width="747" alt="image" src="https://user-images.githubusercontent.com/85433724/204004552-fda94606-c194-4a44-8a1c-0d3c3fe39e9e.png">